### ISSUE TITLE
Query malformed when using refinementlist 

### ISSUE THREAD

**Issue by ElviraCB** [searchkit/searchkit#629]
Hello,

I get a query malformed response from elasticsearch when I make use a RefinementListFilter.
When I remove the RefinementListFilter in my code and search for something I get a valid response back.
> "error":{  
>           "root_cause":[  
>              {  
>                 "type":"illegal_argument_exception",
>                 "reason":"query malformed, empty clause found at [1:39]"
>              }
>           ],
>           "type":"illegal_argument_exception",
>           "reason":"query malformed, empty clause found at [1:39]"
>        },
>        "status":400
> }

**RefinmentListFilter**
```
<RefinementListFilter
                    translations={{ "facets.view_more": "Meer auteurs" }}
                    id="auteur"
                    title="Auteur"
                    field="auteur.raw"
                    operator="OR"
                    size={3}
                  /> 
```
**Mapping:**

```
{
  "producten4" : {
    "aliases" : { },
    "mappings" : {
      "boek" : {
        "properties" : {
          "auteur" : {
            "type" : "text",
            "fields" : {
              "auteur" : {
                "type" : "text"
              },
              "raw" : {
                "type" : "keyword"
              }
            }
          },
          "hoofdtitel" : {
            "type" : "text",
            "fields" : {
              "keyword" : {
                "type" : "keyword",
                "ignore_above" : 256
              },
              "raw" : {
                "type" : "keyword"
              }
            }
          },
        }
      }
    },
    "settings" : {
      "index" : {
        "creation_date" : "1525165241417",
        "number_of_shards" : "5",
        "number_of_replicas" : "1",
        "uuid" : "ENAygLl8QO-5jNRumCc0Tw",
        "version" : {
          "created" : "6020299"
        },
        "provided_name" : "producten4"
      }
    }
  }
}
```
```
<Hits
  hitsPerPage={40}
  highlightFields={[
    "hoofdtitel",
  ]}
  sourceFilter={[
    "titel"
    "auteur",
  ]}
  mod="resultaten-container"
  itemComponent={BookHitsListItem}
/>
```
What am I doing wrong?

**Comment by schermannj** [comment]
@ElviraCB have the same after migration to elasticsearch 6.2

**Comment by ElviraCB** [comment]
I've tried a lot of things to figure out how to get it to work again.

I don't know if it's going to help you out but, I've:

- upgraded Searchkit to ^2.3.0-9, since I was still using 2.2.1-0 before.
- I've reseted my docker, which made me need to rebuild every project, but when I've finished the building it and rebooting my mac, suddenly my filter started to work when I run the project and started to search.
